cmd/cli/plugin/login/main.go

```diff
@@ -354,6 +356,8 @@ func createServerWithEndpoint() (server *configv1alpha1.Server, err error) {
 		isVSphereSupervisor, err := wcpauth.IsVSphereSupervisor(endpoint, getDiscoveryHTTPClient())
 		// Fall back to assuming non vSphere supervisor.
 		if err != nil {
+			log.Fatalf("Error creating kubeconfig with tanzu pinniped-auth login plugin: %v", err)
+			return nil, err
```

Comment:
I am not sure from where this change is coming from. This feels like a change that might not be related to this change. cc @vuil